app/models/bgeigie_import.rb

```diff
@@ -97,7 +97,7 @@ def process
     confirm_status(:compute_latlng)
     update_column(:status, 'processed')
     delete_tmp_file
-    update_column(:version, $LAST_MATCH_INFO[1].to_s) if /^# format=(\d+\.\d+\.\d+)$/m =~ source.file.read
+    update_column(:version, $LAST_MATCH_INFO[1].to_s) if /^# format=(.*?)$/m =~ source.file.read
```

Comment:
I think we can just do a greedy match if we leave out the multi line field. Seems simpler.

Reply:
If I just take the ? out it will grab everything in the file. What else should I do?

Reply:
I think that if you remove the `m` at the end it will no longer match all lines, only to the end of the current line. And dropping the `?` will make it a greedy match. So:

`/^# format=(.*)$/`